### ISSUE TITLE
Timestamps

### ISSUE THREAD

**Issue by ErikKrause** [firephp/firephp-for-firefox-devtools#14]
Timestamps with milliseconds, like in the devtools console, would be nice.

**Comment by cadorn** [comment]
I don't see them. Screenshot? I presume they should show the server time?

**Comment by ErikKrause** [comment]
![zwischenablage01](https://user-images.githubusercontent.com/6303830/32672437-57d08be2-c64b-11e7-991a-7841ddbe4d2a.jpg)
You must disable devtools.webconsole.new-frontend-enabled in FF 57 in order to see this. The output is by my chromelogger fork.

**Comment by cadorn** [comment]
1. Is the time showing the server time?
 2. What timezone is the time? Server timezone or local desktop? We can make it configurable but would be good to use same default.

**Comment by ErikKrause** [comment]
I guess this is a function the devtools console provides, since it is also displayed for CSS, JS etc. events. Hence I assume it is local time. But server time would be better of course, just to see timings between subsequent calls.

**Comment by cadorn** [comment]
Yeah, I agree, **server time** would be better. I don't see client time being useful as all log times would be equal to time of request.

I filed an issue for FirePHPCore here: https://github.com/firephp/firephp-core/issues/46

I see two views for the console once we have server timing info available:

  1) Grouped by request/sub-request and then sorted by index (as is the case now)
  2) Sorted by time across all requests so you can see timing of server events as different requests were processed
